src/examples/basic.py

```diff
@@ -9,14 +9,15 @@ class MyEnvClass(NetworkBase):
     """
 
     def create_hook(self):
-        # Add a bastion host as a child stack in the environment
-        self.add_child_template(Bastion())
+        pass
+        # # Add a bastion host as a child stack in the environment
```

Comment:
I still don't think you should remove all the content from the basic example, that really defeats the purpose of the example